source/tutorials/php.rst

```diff
@@ -1,193 +1,251 @@
+To enable an installed extension, you must add it to the :file:`php.ini` file
+for the composer.
 
-Create the :file:`my-php.ini` file, with the directive to load the php-imagick extension
+#.  Create the :file:`my-php.ini` file using the directive to load the
+    php-imagick extension:
 
-.. code-block:: bash
+    .. code-block:: bash
 
-   sudo echo "extension=imagick.so" >> /etc/php.d/my-php.ini
+       sudo echo "extension=imagick.so" >> /etc/php.d/my-php.ini
 
+#.  Restart the php-fpm service for PHP to pick up the modification to the
+    :file:`/etc/php.d/my-php.ini` file:
 
-No further detail is required to load the extension, but you must restart the php-fpm service for PHP to pick up the modification to the :file:`/etc/php.d/my-php.ini` file.
+    .. code-block:: bash
 
-.. code-block:: bash
+       sudo systemctl restart php-fpm
 
-   sudo systemctl restart php-fpm
+#.  Verify that the imagick extension has been loaded by searching through the
+    runtime list of loaded PHP Modules:
 
-Verify that the imagick extension has been loaded by searching through the runtime list of loaded PHP Modules.
+    .. code-block:: bash
 
-.. code-block:: bash
-
-   php -m | grep imagick
+       php -m | grep imagick
 
 
 .. note::
 
-   To enable an extension, you must install it, add it to the :file:`my-php.ini` file, and restart the :file:`php-fpm` service. However, some extensions may have configuration options, which will be documented by the extension maintainer. Add the options you need to the :file:`/etc/php.d/my-php.ini` file as described in the extension's documentation. Be sure to restart :file:`php-fpm` after changing the file.
+   To enable an extension, you must install it, add it to the
+   :file:`my-php.ini` file, and restart the :file:`php-fpm` service. However,
+   some extensions may have configuration options, which will be documented by
+   the extension maintainer. Add the options you need to the
+   :file:`/etc/php.d/my-php.ini` file as described in the extension's
+   documentation. Be sure to restart :file:`php-fpm` after changing the file.
 
 Configure PHP-FPM
 *****************
 
-The PHP-FPM configuration file is separate from the :file:`php.ini` file used by PHP. |CL| installs the default :file:`php-fpm.conf` file in :file:`/usr/share/defaults/php` and this file will be overwritten with its default values during each software update. However, PHP-FPM requires that the configuration file exist in that location, and by design will not read configuration options from a different path.
-
-One solution to changing PHP-FPM configuration options in |CL| is to manually override the php-fpm.service unit in systemd to pass an explicit location to a custom :file:`php-fpm.conf` file.
+The PHP-FPM configuration file is separate from the :file:`php.ini` file used
+by PHP, however, it has a similar default path restriction. Follow the steps
+below to configure PHP-FPM.
```

Comment:
Again, I want to explain --why-- we do things this way, which is not the "standard" linux method for configuring php-fpm. I want more context here, so people don't get frustrated with  the doc just assuming that they have full context on why there's a tutorial for this.

Reply:
OK. Thanks for sharing your approach.